packages/tools/src/tools/CrosshairsTool.ts

```diff
@@ -137,6 +140,9 @@ export default class CrosshairsTool extends AnnotationTool {
           enabled: false,
           panSize: 10,
         },
+        // actorUIDs for slabThickness application, if not defined, the slab thickness
+        // will be applied to all actors of the viewport
+        actorUIDsForSlabThickness: [],
```

Comment:
I guess this is also a filter right?

Something like: `filterActorUIDsToSetSlabThickness`... or something.